addons/hr_expense/views/hr_expense_views.xml

```diff
@@ -524,15 +530,15 @@
                         <group name="employee_details">
                             <field name="employee_id" context="{'default_company_id': company_id}"/>
                             <field name="payment_mode"/>
-                            <field name="user_id" domain="[('share', '=', False)]"/>
```

Comment:
This goes against https://github.com/odoo-dev/odoo/commit/e428d473c39bbc6fc786ebecb3b7703953579a59

I suppose that this is a rebase issue, as I don't see this point on the spec.